test/csv/parse/test_quote_char_nil.rb

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# frozen_string_literal: false
+
+require_relative "../helper"
+
+class TestCSVParseQuoteCharNil < Test::Unit::TestCase
+  extend DifferentOFS
+
+  def test_general
+    [[%Q{a,b},  ["a", "b"]],
+     [%Q{a,,,}, ["a", nil, nil, nil]],
+     [%Q{,},    [nil, nil]],
+    ].each do |edge_case|
+      assert_equal(edge_case.last, CSV.parse_line(edge_case.first, quote_char: nil))
+    end
+  end
```

Comment:
Could you split these 3 patterns to each test?
We should use one data in one test.

Reply:
Splited into 3. https://github.com/ruby/csv/pull/79/commits/d6ccb1b0a28472e1cba6462fb21b57726e37a9a6